src/CopyButton.js

```diff
@@ -5,13 +5,13 @@ import { useAsyncCallback } from './AsyncButton'
 
 const wait = t => new Promise(res => setTimeout(res, t))
 
-export function useCopyTextHandler(textToCopy, { interval } = {}) {
+export function useCopyTextHandler(textToCopy, { interval, options } = {}) {
```

Comment:
@Arriv9l can we just make options a part of the second param? 

Something like
```suggestion
export function useCopyTextHandler(textToCopy, { interval, ...options } = {}) {
```